src/main.ts

```diff
@@ -205,11 +219,15 @@ async function getActionFile(
       path: 'action.yml'
     })
 
+ 
+
     // todo: warning: duplicated code here
     if ('name' in yml && 'download_url' in yml) {
       result.name = yml.name
       result.owner = repo.owner
       result.repo = repo.name
+      result.forkedfrom = parentinfo
+
```

Comment:
Let's not add extra empty lines, please remove

Reply:
removed extra lines